python/cudf/cudf/tests/test_parquet.py

```diff
@@ -295,6 +299,12 @@ def test_parquet_read_row_group(tmpdir, pdf, row_group_size):
     if "col_category" in gdf.columns:
         gdf = gdf.drop("col_category")
 
+    # pandas.to_parquet converts uint32 to int64 when writing file
```

Comment:
does this mean that `uint32` is not actually being tested here?

Reply:
+1 to this comment, would be good to add a test using pyarrow to make sure we get dtype coverage

Reply:
Pyarrow! Thanks. I wanted to fix this but didn’t get this idea.

Reply:
pyarrow also didn't work.
```python
In [1]: import pandas as pd

In [2]: import pyarrow as pa

In [3]: import pyarrow.parquet as pq

In [4]: import cudf

In [5]: df = pd.DataFrame({'a':pd.Series([1,2,3], dtype='uint32')})

In [6]: padf = pa.Table.from_pandas(df)

In [7]: padf
Out[7]: 
pyarrow.Table
a: uint32

In [8]: pq.write_table(padf, 'pa.parquet')

In [9]: pq.read_table('pa.parquet')
Out[9]: 
pyarrow.Table
a: int64

In [10]: pd.read_parquet('pa.parquet').dtypes
Out[10]: 
a    int64
dtype: object

In [11]: cudf.read_parquet('pa.parquet').dtypes
Out[11]: 
a    int64
dtype: object
```

Reply:
This sounds like a pyarrow bug that we should file then.

Reply:
Could you just add a comment to the JIRA # on Arrow in the test and the Python is good to go from my perspective.

Reply:
[ARROW-9215](https://issues.apache.org/jira/browse/ARROW-9215)